packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -519,6 +520,38 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin
     _drag = null;
   }
 
+  // SCROLL WHEEL
+
+  // Returns the offset that should result from applying [event] to the current
+  // position, taking min/max scroll extent into account.
+  double _targetScrollOffsetForPointerScroll(PointerScrollEvent event) {
+    final double delta = widget.axis == Axis.horizontal
+        ? event.scrollDelta.dx
+        : event.scrollDelta.dy;
+    return math.min(math.max(position.pixels + delta, position.minScrollExtent),
+        position.maxScrollExtent);
+  }
+
+  void _receivedPointerSignal(PointerSignalEvent event) {
+    if (event is PointerScrollEvent && position != null) {
+      final double targetScrollOffset =
+          _targetScrollOffsetForPointerScroll(event);
+      // Only claim the event if it would actually result in a scroll.
+      if (targetScrollOffset != position.pixels) {
+        PointerSignalArenaMember(
```

Comment:
you create an object here but promptly throw it away. If this isn't a bug, then I recommend changing the API. Having a constructor have magical side effects is against this part of our style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-secret-or-global-state.

Reply:
Noted; will fix in the next cycle by pulling `add` out of the constructor.